libgambatte/libretro/libretro.cpp

```diff
@@ -984,6 +984,7 @@ bool retro_load_game(const struct retro_game_info *info)
       { mainram, gb.rambank0_ptr(),  0, 0xC000,               0, 0, 0x1000,                        NULL },
       { mainram, gb.bankedram_ptr(), 0, 0xD000,               0, 0, 0x1000,                        NULL },
       { mainram, gb.zeropage_ptr(),  0, 0xFF80,               0, 0, 0x0080,                        NULL },
+      { mainram, gb.rambank1_ptr(),  0, 0x10000,              0, 0, gb.isCgb() ? 0x7000 : 0,       NULL },
```

Comment:
isnt gb/gbc on has 65536 bytes of memory to use(regardless of bankswitched wram and vram on gbc)? At starting address of 0x10000 where is this pointing to then?

i dont know if there were changes in how cheevos are read if in gbc mode but specifying only 1st bank switched bytes size should be enough.